os/arch/arm/src/armv8-m/up_assert.c

```diff
@@ -479,6 +483,13 @@ void up_assert(const uint8_t *filename, int lineno)
 #else
 	lldbg("Assertion failed at file:%s line: %d\n", filename, lineno);
 #endif
+
+#ifdef CONFIG_DEBUG_WORKQUEUE
+	if (IS_HPWORK || IS_LPWORK) {
+		lldbg("Running work function is %x.\n", work_get_current());
```

Comment:
Is there anyway to know which wq is?

Reply:
In assert log, there is already task name.
I can print the name again, but it is duplicated I think.